bin/build_assets

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env node
+
+const fs = require('fs');
+const fsExtra = require('fs-extra');
+const walk = require('klaw-sync');
+const exec = require('shelljs').exec;
+
+const variants = fs.readdirSync('variants');
```

Comment:
Is there a specific reason you're using fs and fs-extra? I **think** any fs methods are also in fs-extra. If they need to be syncronous, fs-extra should be able to do that as well.